onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -1495,10 +1495,16 @@ def aten_bitwise_xor(self: TInt, other: TInt) -> TInt:
     return op.BitwiseXor(self, other)
 
 
-def aten_blackman_window(window_length: int) -> TensorType:
+def aten_blackman_window(
+    window_length: int,
+    dtype: int = -1,
+    layout: str = "",
+    device: str = "",
+    pin_memory: bool = False,
+) -> TensorType:
     """blackman_window(int window_length, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor"""
 
-    raise NotImplementedError()
+    return op.BlackmanWindow(window_length)
```

Comment:
Can `output_datatype` from the onnx op represent `dtype`?

Reply:
Good catch, thanks!